### PR TITLE
feat(form): display UIForm in drawer

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

Current display of UIForm in drawer is .... not supported
No margin at all, not centered, actions not in the footer, ...

**What is the chosen solution to this problem?**

http://recordit.co/gROevK2RPW

Wrap actions in a div to mimick the Drawer.Footer behavior (max-width 650px)
Add background color

Make all demo support the display in drawer mode:

screen shot 2018-07-16 at 10 18 10

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Comment by annilsson** [comment]
Nice to see this PR, just noticed one thing though:

The scrollbar is hidden behind the footer, and the bottom form field is not fully visible. Maybe flex could solve that instead of position absolute, if possible. I guess that's what @glorieux was thinking as well.

form-in-drawer

File: packages/forms/src/UIForm/UIForm.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:global(.tc-drawer-content) .uiform > :global(.tf-actions-wrapper) {
+	position: absolute;
```
[the remark this answers is not in the document]
Comment (jmfrancois) [reply packages/forms/src/UIForm/UIForm.scss]: I don t understand the question

**Comment by jmfrancois** [comment]
The structure doesn t let me use anything else than position:absolute here.
flex will not work AFAIK

**Comment by jmfrancois** [comment]
re-implement using flex, but with the constraint it not in a drawer but in a `Drawer.Container`

File: packages/forms/stories-core/index.js [hunk 2]
```diff
@@ -32,33 +33,34 @@ const store = createStore(reducer);
...
-		</div>
-	</Provider>
+		</Provider>
 	</I18nextProvider>
 );
 
 const coreConceptsStories = storiesOf('Core concepts', module)
-	.addDecorator(checkA11y)
-	.addDecorator(withKnobs)
+	// .addDecorator(checkA11y)
```
Comment (sgendre) [inline packages/forms/stories-core/index.js]: are those 2 lines kept on purpose?

**Pull request merged**